packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -30,42 +36,46 @@ export const isMain = (meta: ImportMeta) => {
   return process.argv[1] === fileURLToPath(meta.url);
 };
 
-export function populateOutputBundleOptions(cwd: string): OutputBundleOptions {
-  const outputBundleDir = join(cwd, ".apphosting");
+export function populateOutputBundleOptions(rootDir: string, appDir: string): OutputBundleOptions {
+  const outputBundleDir = join(rootDir, ".apphosting");
```

Comment:
can we make sure to add some javadocs for the params here too? What is a rootdir, what is an appdir?

In particular, I'm surprised that we're looking to write .apphosting to the rootdir. It would seem more appropriate to write it to the same levels as the application directory.

If in the future we support building multiple backends in one build process, writing it to the rootdir will cause subsequent builds to stomp on previous build artifacts